### PR TITLE
Add SulinOS Installation Steps

### PR DESCRIPTION
**Pull request by sulincix**
SulinOS install guide added.

### CODE REVIEW

**Comment by sulincix** [comment]
hmmm
we need fix

**Comment by sulincix** [comment]
photo_2020-04-19_12-31-41
flatpak works well

**Comment by frknkrc44** [comment]
Looks good 😋 please merge it

Because SulinOS is an independent base GNU/Linux distro and current installation ways don't compatible with this distro

**Comment by paledega** [comment]
Nice
SulinOS is good distro.

**Comment by AsciiWolf** [comment]
Is the graphical installation of Flatpaks supported? If not, you should add `<p>Note: graphical installation of Flatpak apps may not be possible with SulinOS.</p>` (just like, for example, Gentoo).

**Comment by AsciiWolf** [comment]
By the way, just wondering: How big is the SulinOS user base? To be honest, this is the first time I heard about this distro.

**Comment by mwleeds** [comment]
The CI is failing in strange ways on this.

**Comment by paledega** [comment]
> Is the graphical installation of Flatpaks supported? If not, you should add `<p>Note: graphical installation of Flatpak apps may not be possible with SulinOS.</p>` (just like, for example, Gentoo).

Sulin does not have any graphical installer. (may we will create in next times)

**Comment by paledega** [comment]
> By the way, just wondering: How big is the SulinOS user base? To be honest, this is the first time I heard about this distro.

SulinOS indipendent distro. Uses inary package manager and Openrc. 
Https://sulinos.github.io

**Comment by paledega** [comment]
> The CI is failing in strange ways on this.

what we can do

File: data/distro.yml [hunk 1]
```diff
@@ -476,3 +476,30 @@
...
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
```
Comment (AsciiWolf) [inline data/distro.yml:502]: Needs information about graphical installation not being possible, see: https://github.com/flatpak/flatpak.github.io/pull/374#issuecomment-616483391

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
> SulinOS indipendent distro. Uses inary package manager and Openrc.
> [Https://sulinos.github.io](Https://sulinos.github.io)

And how big (approximately) is its user base? :-)

**Comment by sulincix** [comment]
SulinOS is turkey origin distro. Most of turkish users may use sulinos. (I never track usage)
But our telegram community group has 500+ member (Turkish group)
Hovewer SulinOS is global distro. Non-turkish users can use sulinos. 


Graphical install not possible yet. I will add this sentence in pull request.
Thanks for attention

**Review by AsciiWolf — APPROVED** [review]

**Comment by paledega** [comment]
now We need to fix CI error.

**Comment by barthalion** [comment]
` Loading data file "/github/workspace/data/distro.yml" failed due to an error: (/github/workspace/data/distro.yml): did not find expected '-' indicator while parsing a block collection at line 1 column 1`

Your changes are probably indented wrongly. Compare it with other entires in distro.yml file.

**Comment by sulincix** [comment]
I found bug and I fixed it.

**Pull request merged**